routes/signout.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (function() {
+  var express = require('express');
+  var router = express.Router();
+
+//Signout
+  router.get('/', function(req, res){
+    req.session.signedin = false;
```

Comment:
Would setting signedin to false be enough? You might need to reset the whole session object to clear username, id, and active for security purposes.

Reply:
Use "req.session.destroy();" instead.

Reply:
fixed to destroy method instead.